evm/its.js

```diff
@@ -678,8 +667,41 @@ async function processCommand(config, chain, options) {
                 gasValue,
                 gasOptions,
             );
+            await handleTx(tx, chain, interchainTokenService, action, 'LinkTokenStarted');
+            break;
+        }
+
+        case 'set-trusted-chain': {
+            const [trustedChain] = args;
+            const owner = await new Contract(interchainTokenService.address, IOwnable.abi, wallet).owner();
 
-            await handleTx(tx, chain, interchainTokenService, options.action, 'LinkTokenStarted');
+            if (owner.toLowerCase() !== walletAddress.toLowerCase()) {
+                throw new Error(`${action} can only be performed by contract owner: ${owner}`);
+            }
+
+            validateParameters({ isNonEmptyString: { trustedChain } });
+
+            const trustedChainFinal =
+                getChainConfig(config, trustedChain.toLowerCase(), { skipCheck: true })?.axelarId || trustedChain.toLowerCase();
+            const trustedAddressFinal =
+                'hub' || getChainConfig(config, trustedChain.toLowerCase())?.contracts?.InterchainTokenService?.address;
+
+            if (trustedChainFinal === undefined || trustedAddressFinal === undefined) {
+                throw new Error(`Invalid chain/address: ${trustedChain}`);
+            }
+
+            const trustedChains = [trustedChainFinal];
+            const trustedAddresses = [trustedAddressFinal];
+
+            if (prompt(`Proceed with setting trusted address for chain ${trustedChains} to ${trustedAddresses}?`, yes)) {
+                return;
+            }
+
+            for (const [trustedChain, trustedAddress] of trustedChains.map((chain, index) => [chain, trustedAddresses[index]])) {
```

Comment:
ITS has multicall, which avoids the need to submit multiple txs

Reply:
This PR should really be two separate PRs. One PR to refactor existing cmds, and another to add a new cmd on top. This makes it easier to review and merge independently